components/typography/styles.js

```diff
@@ -318,3 +318,65 @@ export const blockquoteStyles = css`
 		}
 	}
 `;
+
+const importUrl = 'https://s.brightspace.com/lib/fonts/0.6.1/assets/';
```

Comment:
A couple notes about the exports...

* `importUrl` and `fonts` was previously exported but they are now unused. They were originally added by Allan for the preload font tests, which were removed when the new testing library was introduced.
*  `fontFacesCss` is include in `styles.js` rather than `typography.js` so that `BrightspaceUI/typography` can import them without mutating the DOM.